exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -165,7 +165,7 @@ table.sortable thead .sorting_desc { background-image: url("/static/img/black-de
   <#assign queued = queueName != "" && queueName != "-" />
 
   <div class="page-header"></div>
-  <h3>Query Profile</h3>
+  <h3>Query Profile : <span style='font-size:85%'>${model.getQueryId()}</span></h3>
```

Comment:
what does this return ? text ?

Reply:
Yes. The backing ProfileWrapper returns the queryID as a text.